### PR TITLE
Working config by default

### PR DESCRIPTION
**Pull request by jkrems**
These changes make sure that our linting config works "by default",
e.g. w/o having to install multiple packages into a project.

All it needs:

```bash
npm i -D eslint eslint-config-groupon
echo '{"extends":"groupon/base"}' > .eslintrc
```

### CODE REVIEW

File: linters/eslint-config-groupon/legacy.js [hunk 1]
```diff
@@ -25,12 +25,31 @@ NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
...
+    require.resolve('eslint-config-airbnb/rules/strict'),
+    require.resolve('eslint-config-airbnb/rules/style'),
+    require.resolve('eslint-config-airbnb/rules/variables')
   ],
-  rules: {
+  'env': {
+    'browser': true,
+    'node': true,
+    'amd': false,
+    'mocha': false,
```
Comment (khoomeister) [inline linters/eslint-config-groupon/legacy.js]: Does this mean mocha can't be used?

**Comment by khoomeister** [comment]
Aside from my question, :shipit:

File: linters/eslint-config-groupon/legacy.js (hunk 1 above)
Comment (jkrems) [inline linters/eslint-config-groupon/legacy.js]: It just means that it won't be on-by-default. You can still set `env: { mocha: true }` in `test/.eslintrc`. That's what I did for `testium-driver-wd` and it seems to work.

Comment (khoomeister) [inline linters/eslint-config-groupon/legacy.js]: OK cool!

**Pull request merged**